conf/parser.py

```diff
@@ -147,6 +149,16 @@ def parse(self, ifaces):
             print('Can\'t parse interface name(s)! Setting it to default values ({}, {})'.format(
                 "access", "core"))
 
+        # UnixPort Paths
+        try:
+            self.notify_sockaddr = self.conf["notify_sockaddr"]
+            self.endmarker_sockaddr = self.conf["endmarker_sockaddr"]
```

Comment:
wonder if the second one is given but not the first one, this might be inconsistent with the user wanted?

Reply:
done